web_src/js/features/repo-issue-sidebar.ts

```diff
@@ -265,10 +267,12 @@ export function initRepoIssueSidebar() {
   initListSubmits('select-label', 'labels');
   initListSubmits('select-assignees', 'assignees');
   initListSubmits('select-assignees-modify', 'assignees');
+  initListSubmits('select-reviewers', 'reviewers');
   initListSubmits('select-reviewers-modify', 'assignees');
```

Comment:
I have resolved some conflicts. But here I haven't read it carefully.

This PR ever changed it to `initListSubmits('select-reviewers-modify', 'reviewers');`

Was it done intentionally? If yes, what's the reason behind it?

Reply:
Hi, the class for reviewer list has changed to `ui reviewers list` so I was expecting the `initListSubmits` should init that list too. 

https://github.com/CalK16/gitea/blame/941f7fdf806c706af3bcc5e3d4960cb4a0856a1f/templates/repo/issue/new_form.tmpl#L89C17-L89C17

Reply:
I do not think changing `select-reviewers-modify` is right. I do not see any change related to `select-reviewers-modify`

Reply:
You are probably right. I double checked the `select-reviewers-modify` and found that it only applied to the sidebar for a created pull request. And that is distinguished from the `select-reviewers`.While this is true, I think I've done something wrong based on my previous understanding. Allow me to spend sometime working on it. Thanks for the correction.

Reply:
Let's have this first:  Split issue sidebar into small templates #32444 , to have a clear codebase and then continue.